specs/labels-and-localization/README.md

```diff
@@ -0,0 +1,306 @@
+
+Cons
+- More verbose than putting the APIs directly on nimble-theme-provider (but that's not really extensible)
+- Still somewhat verbose in the page, to have all of the localized strings as attributes
+    - We can consider using the `fromView` attribute mode on the attributes for the strings on the theme-provider. If clients set the strings using the theme-provider properties rather than as attributes in the HTML (as is currently proposed for Angular), `fromView` would not reflect those strings back to the DOM.
+- Apps may be pulling in more strings than they need (i.e. if they just use the banner, they'll pick up the rest of the strings in `i18n-core`). Mitigation is to split off large sets of strings (e.g. those for the table).
+- Not an originally intended use case for DesignToken
+
+Other notes:
+- We don't expect to need mixed content (i.e. other than simple strings), so attributes should be sufficient (vs. slots)
+- If the page was automatically translated by something like Google Translate, the attributes on the i18n providers don't get translated. However we think this is OK because the expected usages of the labels (button content, slotted content, etc) would get translated.
+- We may want to provide a description along with each English string, to aid in translation.
+
+The current set of known labels for Nimble is shown below:
+
+**nimble-i18n-core**  
+| Token Name             | English string |
+|------------------------|----------------|
+| banner-dismiss         | Close         |
+| number-field-increment | Increment     |
+| number-field-decrement | Decrement     |
+
+**nimble-i18n-table**  
+| Token Name                            | English string      |
+|---------------------------------------|---------------------|
+| table-group-collapse                  | Collapse group      |
+| table-group-expand                    | Expand group        |
+| table-groups-collapse-all             | Collapse all groups |
+| table-cell-action-menu-label          | Options             |
+| table-column-header-menu              | Column Options      |
+| table-column-header-grouped-indicator | Grouped             |
+| table-column-sort-ascending           | Sort ascending      |
+| table-column-sort-descending          | Sort descending     |
+| table-column-group-by                 | Group by            |
+| table-column-size-to-content          | Size to content     |
+
+Note: We will probably remove the `table` prefix from the properties and attribute names on `nimble-i18n-table` as it's redundant. `table-cell-action-menu-label` is a fallback for when column.actionMenuLabel is unset.
+
+The expected format for token names is:
+- component name, e.g. `number-field` or `table`
```

Comment:
I'll revamp the guidance to be closer to what we have for regular design tokens.
Since we're not doing a separate i18n provider for each control I think we can support multiple components using the same label if it's for the same concept.